content/src/Environment.ts

```diff
@@ -357,6 +360,16 @@ export class EnvironmentBuilder {
       () => process.env.REPOSITORY_QUEUE_MAX_QUEUED ?? 50
     )
 
+    /*
+     * These are configured as 'CACHE_{CACHE_NAME}_{ENTITY_TYPE}=MAX_SIZE'.
+     * For example: 'CACHE_ENTITIES_BY_POINTERS_SCENE=1000
+     */
+    this.registerConfigIfNotAlreadySet(
+      env,
+      EnvironmentConfig.CACHE_SIZES,
+      () => new Map(Object.entries(process.env).filter(([name]) => name.startsWith('CACHE')))
```

Comment:
By default the caches are not being used, right?

Reply:
Yes, they are being used. This is just to be able to override the max sizes of the caches